lib/index.js

```diff
@@ -484,7 +486,7 @@ export async function exec(argv2) {
   log.debug('Targets:', JSON.stringify(targets, null, 2));
 
   for (const target of targets) {
-    if (await exists(target.output)) {
+    if (fs.existsSync(target.output)) {
```

Comment:
fs not defined here ?

Reply:
Sorry I screw up , It was a bad idea to try  fix the "deprecated fs.exits" in this PR , I will address this in a separate PR.

Reply:
I've fixed this in my TypeScript PR 😄